packages/react/src/checkbox/icons-svg/IconChecked.tsx

```diff
@@ -0,0 +1,19 @@
+import * as styles from "../Checkbox.css";
+export const IconChecked = (
+  <svg
+    {...styles.iconChecked()}
+    fill="none"
```

Comment:
As its need to be controlled by css

```suggestion
    fill="currentColor"
```